src/lib/cost_model/cost_model_logical.hpp

```diff
@@ -10,8 +10,14 @@ class AbstractExpression;
  * Cost model for logical complexity, i.e., approximate number of tuple accesses
  */
 class CostModelLogical : public AbstractCostEstimator {
- protected:
-  Cost _estimate_node_cost(const std::shared_ptr<AbstractLQPNode>& node) const override;
+ public:
+  using AbstractCostEstimator::AbstractCostEstimator;
+
+  std::shared_ptr<AbstractCostEstimator> clone_with_caches(
```

Comment:
overrides aren't usually doc'ed, declaration in base class has this doc:

```
  /**
   * @return a new instance of this estimator with a new instance of the wrapped cardinality estimator, both with mint
   *         caches. Used so that caching guarantees can be enabled on the returned estimator.
   */
```